node/libs/protobuf/src/build.rs

```diff
@@ -0,0 +1,85 @@
+//! Build-related functionality. This should only be used by the code generated by the `protobuf_build` crate.
+
+pub use once_cell::sync::Lazy;
+pub use prost;
+use prost::Message as _;
+pub use prost_reflect;
+use prost_reflect::prost_types;
+use std::sync::RwLock;
+
+/// Global descriptor pool.
+static POOL: Lazy<RwLock<prost_reflect::DescriptorPool>> = Lazy::new(RwLock::default);
+
+/// Protobuf descriptor + info about the mapping to rust code.
+#[derive(Debug)]
+pub struct Descriptor(());
+
+impl Descriptor {
+    /// Constructs a descriptor and adds it to the global pool.
+    pub fn new(_dependencies: &[&'static Self], descriptor_bytes: &[u8]) -> Self {
```

Comment:
if unused, please remove

Reply:
Having dependency refs [allows ensuring](https://github.com/matter-labs/era-consensus/blob/cd08b9cb49f11dad67c5f73e02c2083fe8a921ae/node/libs/protobuf/src/build.rs#L23) via type system that these dependencies are added in the global pool. Would it be better to force dependencies to get initialized via [`Lazy::force()`](https://docs.rs/once_cell/latest/once_cell/sync/struct.Lazy.html#method.force), or in some other way?

Reply:
Type system doesn't ensure any relation between the dependencies and the newly constructed descriptor, so this is as good as Lazy::force (or just dereferencing the lazy descriptors).